### PR TITLE
fix(v2): update Infima website URL

### PR DESCRIPTION
**Pull request by yangshun**
## Motivation

Change Infima website to the WIP component site

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Preview

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

**Comment by yangshun** [comment]
My bad -_-

**Comment by yangshun** [comment]
I'll sneak that fix in later. Luckily GitHub pages does HSTS.

File: website/docs/styling-layout.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Any CSS you write within that file will be available globally and can be referen
 
 ## Styling your site with Infima
 
-`@docusaurus/preset-classic` uses [Infima](https://infima-dev.netlify.com/) as the underlying styling framework. Infima provides flexible layout and common UI components styling suitable for content-centric websites (blogs, documentation, landing pages). For more details, check out the [Infima website](https://infima-dev.netlify.com/).
+`@docusaurus/preset-classic` uses [Infima](http://facebookincubator.github.io/infima/) as the underlying styling framework. Infima provides flexible layout and common UI components styling suitable for content-centric websites (blogs, documentation, landing pages). For more details, check out the [Infima website](http://facebookincubator.github.io/infima/).
```
Comment (ykzts) [inline website/docs/styling-layout.md]:
```suggestion
`@docusaurus/preset-classic` uses [Infima](https://facebookincubator.github.io/infima/) as the underlying styling framework. Infima provides flexible layout and common UI components styling suitable for content-centric websites (blogs, documentation, landing pages). For more details, check out the [Infima website](https://facebookincubator.github.io/infima/).
```

`http`?

Comment (yangshun) [reply website/docs/styling-layout.md]: Yeah we plan to change that to https later. Surprisingly the build's URL is https o.o

See link here - https://v2.docusaurus.io/docs/styling-layout/#styling-your-site-with-infima

Comment (ykzts) [reply website/docs/styling-layout.md]: It's strange but right :smile:

Comment (endiliey) [reply website/docs/styling-layout.md]: I patched it here https://github.com/facebook/docusaurus/pull/1955

**Pull request merged**